src/zjs_ble.c

```diff
@@ -492,7 +492,7 @@ static void zjs_ble_connected(struct bt_conn *conn, uint8_t err)
         char addr[BT_ADDR_LE_STR_LEN];
         bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
         ble_conn.default_conn = bt_conn_ref(conn);
-        zjs_signal_callback(ble_conn.connected_cb_id, &addr, sizeof(addr));
+        zjs_signal_callback(ble_conn.connected_cb_id, addr, sizeof(addr));
```

Comment:
They aren't even the same, addr on its own casts to "pointer to type of element", while type of &addr is "pointer to array of elements, of given size". Difference is negligible for C, but C++ would start to complain here...